external/storm-hdfs/src/test/java/org/apache/storm/hdfs/spout/TestHdfsSpout.java

```diff
@@ -772,7 +773,7 @@ public MockTextFailingReader(FileSystem fs, Path file, Map<String, Object> conf)
         private final int componentId;
 
         public MockTopologyContext(int componentId, Map<String, Object> topoConf) {
-            super(null, topoConf, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null, null);
+            super(null, topoConf, null, null, null, null, null, null, null, 0, 0, null, null, null, null, null, null, null);
```

Comment:
TopologyContext throws an exception if you give it null task id now. I think it is weird to give it null for Integer-type variables, so I've just set both to null.

Reply:
To clarify, the TopologyContext change was made in an earlier PR, but since the tests weren't running, we weren't seeing the exception.